.github/workflows/test-libcxx-runner-infa.yaml

```diff
@@ -0,0 +1,35 @@
+# A sanity check for the libc++ github actions runners. It simply attempts to schedule a job on each runner and
+# then prints some info about each runner.
+#
+# The runner sets are currently:
+#   - libcxx-runners-8-set
+#   - libcxx-runners-32-set
+
+# FIXME: Write this workflow
```

Comment:
does this mean that this workflow is not complete, or is this just a forgotten comment.